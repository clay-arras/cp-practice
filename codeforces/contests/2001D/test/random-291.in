1
24
1 18 21 18 7 5 22 16 17 17 1 18 17 24 9 22 20 3 25 10 21 21 4 8
