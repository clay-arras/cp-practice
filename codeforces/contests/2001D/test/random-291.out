16
21 7 5 16 17 1 18 24 9 22 20 3 25 10 4 8
